shortfin/README.md

```diff
@@ -1,20 +1,44 @@
 # shortfin - SHARK inference library and serving engine
 
-## Simple User Installation
+The shortfin project is SHARK's open source, high performance inference library
+and serving engine. Shortfin consists of these major components:
 
-Install:
+* The "libshortfin" inference library written in C/C++ and built on
+  [IREE](https://github.com/iree-org/iree)
+* Python bindings for the underlying inference library
+* Example applications in
+  ['shortfin_apps'](https://github.com/nod-ai/SHARK-Platform/tree/main/shortfin/python/shortfin_apps)
+  built using the python bindings
 
+## Prerequisites
+
+* Python 3.11+
+
+## Simple user installation
+
+Install the latest stable version:
+
+```bash
+python -m pip install shortfin
 ```
+
+## Developer guides
+
+### Install local packages
+
+After cloning this repository, from the `shortfin/` directory:
+
+```bash
 python -m pip install .
```

Comment:
```suggestion
python -m pip install -e .
```
Editable?

Reply:
Sure. Reworked into a "quick start" step and added `python -m pip install -r requirements-tests.txt`.